tests/bluetooth/host/id/bt_id_read_public_addr/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+
+FILE(GLOB SOURCES src/*.c)
+
+project(bt_id_read_public_addr)
+add_compile_definitions(test_unit_name="${PROJECT_NAME}")
```

Comment:
Don't do this. Instead, spell out the function name in the messages where this is used.

Reply:
Most of the error messages are the same for different tests except that they differs in the API under testing.
I was looking for a way to unify the printed messages structure.

Ok, I will remove such definition from `CMakeLists.txt` files and use `__func__` which combines the test suite and test case names.

Reply:
You can also use some generic form like "Got incorrect return value %d". Or "It incorrectly returned %d". The name of the test is printed above the error during the test run anyway, so adding `__func__` to the error string does not carry any extra information.